client/ngapp/scripts/controllers/todo.js

```diff
@@ -137,23 +146,18 @@ angular.module('loopbackExampleFullStackApp')
   });
 
   $scope.resolveUsingSource = function(conflict) {
-    conflict.resolve(refreshConflicts);
+    conflict.resolveUsingSource(refreshConflicts);
   };
 
   $scope.resolveUsingTarget = function(conflict) {
-    if(conflict.targetChange.type() === 'delete') {
-      conflict.SourceModel.deleteById(conflict.modelId, refreshConflicts);
-    } else {
-      var m = new conflict.SourceModel(conflict.target);
-      m.save(refreshConflicts);
-    }
+    conflict.resolveUsingTarget(refreshConflicts);
   };
 
   $scope.resolveManually = function(conflict) {
-    conflict.manual.save(function(err) {
-      if(err) return errorCallback(err);
-      conflict.resolve(refreshConflicts);
-    });
+    conflict.resolveManually({
```

Comment:
I figured this out by digging into your source code, but we definitely need some actual docs explaining how to resolve conflicts manually.